### PR TITLE
feat: rules page added

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
added fully responsive quiz rule page

### CODE REVIEW

**Review by fahd23 — APPROVED** [review]
Good to merge

File: rules/rules.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+.rules p {
+    padding: 1rem;
+}
+
+.rules p:last-child {
+    text-align: center;
+}
+
+.rules p:nth-child(7) {
+    text-align: center;
```
Comment (paarthbajaj) [inline rules/rules.css:26]: You can convert this into generic utility class and use it whenever you want.

File: rules/rules.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    border-radius: .5rem;
+}
+
+h2 {
+    text-align: center;
+    padding: 1rem;
+}
+
+.rules p {
+    padding: 1rem;
```
Comment (paarthbajaj) [inline rules/rules.css:18]: You can use utility class here as well.

**Review by paarthbajaj — APPROVED** [review]
You can merge just read my comments once.

File: rules/rules.css (hunk 1 above)
Comment (SuyashTiwariCoder) [reply rules/rules.css:26]: ok i will do it next time

File: rules/rules.css (hunk 2 above)
Comment (SuyashTiwariCoder) [reply rules/rules.css:18]: noted

**Pull request merged**